### PR TITLE
logging StateChange/Event JSON encoded

### PR DESCRIPTION
**Pull request by hackaugusto**
This will make it easier to parse the data and use tools like `jq`.
Additionally it will always contain all the necessary data, in in a
machine/human readable format.

### CODE REVIEW

File: raiden/raiden_service.py [hunk 1]
```diff
@@ -413,15 +434,23 @@ def get_block_number(self):
         return views.block_number(self.wal.state_manager.current_state)
 
     def handle_state_change(self, state_change):
-        log.debug('STATE CHANGE', node=pex(self.address), state_change=state_change)
+        log.debug(
+            'STATE CHANGE',
```
Comment (palango) [inline raiden/raiden_service.py]: Could we change this to non-CAPS?

File: raiden/raiden_service.py [hunk 2]
```diff
@@ -413,15 +434,23 @@ def get_block_number(self):
...
 
         event_list = self.wal.log_and_dispatch(state_change)
 
         if self.dispatch_events_lock.locked():
             return []
 
         for event in event_list:
-            log.debug('RAIDEN EVENT', node=pex(self.address), raiden_event=event)
+            log.debug(
+                'RAIDEN EVENT',
```
Comment (palango) [inline raiden/raiden_service.py]: Same here.

File: raiden/raiden_service.py [hunk 3]
```diff
@@ -59,6 +59,27 @@
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def _redact_secret(data):
```
Comment (palango) [inline raiden/raiden_service.py]: Same discussion as before: Didn't we say we don't want to do this as these are possibly  big objects?

Comment (hackaugusto) [reply raiden/raiden_service.py]: This will probably be used with small objects (state changes and events).

**Pull request merged**